app/models/featured_profile.rb

```diff
@@ -0,0 +1,15 @@
+class FeaturedProfile < ApplicationRecord
+  translates :title, :description
+  globalize_accessors :locales => [:de, :en], :attributes => [:title, :description]
+
+  def self.featured_women
+    featured = FeaturedProfile.find_by(public: true)
+    if featured == nil
+      return nil
+    else
+      featured_profiles = []
+      featured.profile_ids.each { | id | featured_profiles << Profile.find(id) }
```

Comment:
Now when an id is wrong and not in the database you get an error on the startpage. we should make sure that the startpage is still working.

Sth like `if Profile.find(id)` ?

Reply:
thanks for suggestion, I add this to the next commit